cmd/bosun/conf/conf.go

```diff
@@ -295,6 +295,31 @@ func (ns *Notifications) Get(c *Conf, tags opentsdb.TagSet) map[string]*Notifica
 	return nots
 }
 
+func GetNotificationChains(c *Conf, n map[string]*Notification) ([][]string) {
```

Comment:
this needs a comments about exactly what it returns. I'm kinda confused on the myself. A list of lists of chains for a single alert? With no duplicates. Maybe writing it out will help.